src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -487,7 +491,71 @@ private void updateBundleAndRecipientListWithResponses(
         bundle.put(question, responses);
         recipientList.put(question.getId(), recipients);
     }
-    
+
+    /**
+     * Removes instructors who are not displayed to students from the
+     * {@link FeedbackQuestionBundle}
+     * 
+     * @param bundle
+     *            {@link FeedbackQuestionsBundle} containing questions for the
+     *            session
+     */
+    private void filterOutHiddenInstructors(FeedbackSessionQuestionsBundle bundle) {
```

Comment:
@unyoungwax, btw, can we make this more similar to how students are filtered out for the instructors' submission page? The way we handle visibility for specific students/instructors is not very standardised now, but let's try to make them similar.